moses/latentgan/model.py

```diff
@@ -0,0 +1,226 @@
+import torch.nn as nn
```

Comment:
Remove pretrained model files—they weight too much. You can put them on google drive and provide a link to them in README for LatentGAN

Reply:
I think we'll deposit trained models in a release section on GitHub later